kobo/settings/base.py

```diff
@@ -695,3 +704,7 @@ def __init__(self, *args, **kwargs):
 MONGO_CONNECTION = MongoClient(
     MONGO_CONNECTION_URL, j=True, tz_aware=True, connect=False)
 MONGO_DB = MONGO_CONNECTION[MONGO_DATABASE['NAME']]
+
+
+SESSION_ENGINE = "redis_sessions.session"
+SESSION_REDIS = RedisHelper.config(default="redis://redis_cache:6380/2")
```

Comment:
Make sure that `redis_cache` has persistent storage

Reply:
It does. 
From `docker-compose.backend.template.yml` in `kobo-docker` related branch. 

```
  redis_cache:
    image: redis:3.2
    # Map our "cache" Redis config into the container.
    volumes:
      - ./.vols/redis_cache_data/:/data/
```